lib/static/styles.css

```diff
@@ -143,19 +143,29 @@
     display: block;
 }
 
-.image-box__screenshot {
+.image-box__screenshot-container {
     max-width: 100%;
+    position: relative;
 }
 
-.image-box__screenshot_auto-size {
+.image-box__screenshot-container_auto-size {
     height: auto;
 }
 
-.image-box__screenshot_fixed-size {
+.image-box__screenshot-container_fixed-size {
     display: table-cell;
     background-size: contain;
     background-repeat: no-repeat;
 }
+
+.image-box__screenshot {
+    position: absolute;
```

Comment:
We can't set the image inside the div-container in a `relative` way due to `padding-top` using. `Padding-top` shifts the image to the bottom, if it is set in a relative way. And we use `padding-top` as a mechanism that allows us to get height proportional to width (saving aspect ratio) of our image. See https://css-tricks.com/aspect-ratio-boxes/